packages/workers-shared/asset-server-worker/wrangler.toml

```
@@ -0,0 +1,3 @@
+name = "asset-server"
+main = "src/index.ts"
+compatibility_date = "2021-11-02" # Oldest version of the runtime
```

```suggestion
compatibility_date = "2024-07-31"

```

Pages' ASW is currently a bit stuck on old stuff, so let's start off with our best foot forwards with this new one if we can.

---

yeah, I'm glad you caught this! I meant to ask if this is indeed the campat_date we want to commit to